homeassistant/components/dsmr/sensor.py

```diff
@@ -527,14 +540,14 @@ class DSMREntity(SensorEntity):
     entity_description: DSMRSensorEntityDescription
     _attr_has_entity_name = True
     _attr_should_poll = False
+    telegram: dict[str, DSMRObject] | None = {}
```

Comment:
I'd try to avoid initializing the class attribute to a mutable object, unless that's specifically what we want. Mutable class attributes are error prone.

Reply:
Oh right 🤒 

this thing needs a coordinator 🙈

Reply:
Done